### PR TITLE
docs(README): note project is no longer actively maintained

### PR DESCRIPTION
**Pull request by mikehardy**
Per https://github.com/hieuvp/react-native-fingerprint-scanner/issues/178#issuecomment-831643755

Not sure if there is anything else you want to put in here?

I can think of two things that would be useful at least, but everything's optional it's open source right?

- how you'll hand over the repo:

Typically it would be via transferring ownership (so issues / PRs stay intact). Highest value
Or you could transfer to a new organization and then add new owners, or perhaps you are simply
done with it and the suggestion is "just fork it and carry on, I'll add a comment if people adopt your repo"

- how you'd prefer to be contacted for someone that wants to take it over
 
an issue asking, or a couple accepted PRs or whatever

Either way cheers - you have a popular repo and quite a few users, totally natural to move on

For my part, if you created an organization ('react-native-fingerprint-scanner' perhaps, with 'react-native-fingerprint-scanner' in it) where you were owner, and you added me as a role that could add collaborators I would not commit to programming time on it but I would respond to issues / merge reasonable+obvious things for forwards-compatibility and petition all issue/PR participants to become contributors to keep it alive. Just enough so it was alive for the next motivated maintainer essentially

### CODE REVIEW

**Comment by phillbaker** [comment]
👋 hey @mikehardy, sorry for not seeing this sooner. Thanks for the offer of help! I don't have admin permissions on this project, so I can't add you as a maintainer or do any of the transfers suggested. @hieuvp would have to help out on that.

**Comment by hieuvp** [comment]
Hi @mikehardy, thank you for being supportive in this open source project so far, I have added you into maintainer list.
If you have any other suggestion, please let me know.

**Comment by mikehardy** [comment]
Ah great - I've got a few other things cooking at the moment, however, in general breathing new life into a repo involves a general idea of "making maintenance as easy as possible, and get a rhythm of releases going for users"

- automated semantic release is then a #1 priority so that maintenance and release is reduced to PR review, the real mental work of maintenance
- automated CI then becomes more important to automate PR review as much as possible
- honestly, that's about it

So the next time I pass through here, or if anyone wants to propose a PR (:pray: !) I'll see about documenting current release process and adding a semantic release bot like exist in lots of react-native repos, 

@hieuvp if you have not granted me access to edit repo secrets + add a bot user, that will  be necessary in order to configure automated semantic release

File: README.md [hunk 1]
```diff
@@ -1,5 +1,12 @@
 # React Native Fingerprint Scanner
 
+# Looking for Maintainers
+
+This project is no longer actively maintained
```
Comment (mikehardy) [inline README.md:5]:
```suggestion
This project is no longer actively maintained by the previous maintainers.
If you would like to propose a PR we can merge it though, it needs your effort to continue!
```

**Comment by hieuvp** [comment]
Hey @mikehardy, just sent you a private email.
I am happy to connect.

**Pull request merged**